packaging/installer-linux/installer-debian/src/main/resources/common/neo4j

```diff
@@ -46,6 +47,14 @@ SCRIPTNAME=/etc/init.d/$NAME-service
 
 do_start()
 {
+  # Make sure that /var/run/neo4j exists
```

Comment:
I am generally against "narrative" comments like this. I'd expect anyone reading the file to be able to tell that that's what the line below does.